### PR TITLE
Line edit

### PR DESCRIPTION
**Pull request by ankita240796**
addresses #96 

## What is included in this PR?
Minor line edits for docs.

### CODE REVIEW

File: aggregate-limitations.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@ Aggregate Limitations
    If you have many media attachments or more than 10MB of media attachments, :doc:`Form Uploader <form-uploader>` provides a quick and easy means to upload forms into ODK Aggregate.
 
 
-- ODK Aggregate 1.4.7 and earlier use a deprecated `backends` technology. Google may terminate support for that at any time with little warning. You should consider upgrading to 1.4.8.  
+- ODK Aggregate 1.4.7 and earlier use a deprecated *backends* technology. Google may terminate support for that at any time with little warning. You should consider upgrading to 1.4.8.
```
Comment (adammichaelwood) [inline aggregate-limitations.rst]: I think that backend is a simple adjective here, and should have:

 - no 's'
 - no markup

> ...use a deprecated backend technology.

File: aggregate-upgrade.rst [hunk 2]
```diff
@@ -164,7 +164,7 @@ Aggregate v1.4.12
...
 5. Tweak Google AppEngine configuration to reduce frontend usage hours and thereby reduce costs.
 6. Performance improvements to Google AppEngine task locks (primarily for ODK 2.0 support).
-7. Add "/users/list" URL that returns the list of all configured users and the roles they have been granted. Access to this URL requires authentication and is restricted to ODK Aggregate usernames and Google accounts (anonymousUser access is forbidden). If the authenticated user does not have Tables Super-user, Administer Tables, or Site Administrator permissions, a singleton list is returned that contains only information about their own identity and its capabilities.
+7. Add **/users/list** URL that returns the list of all configured users and the roles they have been granted. Access to this URL requires authentication and is restricted to ODK Aggregate usernames and Google accounts (anonymousUser access is forbidden). If the authenticated user does not have Tables Super-user, Administer Tables, or Site Administrator permissions, a singleton list is returned that contains only information about their own identity and its capabilities.
```
Comment (adammichaelwood) [inline aggregate-upgrade.rst]: If that is a literal URL string, it should be in double-backticks.

File: aggregate-upgrade.rst [hunk 3]
```diff
@@ -194,8 +194,8 @@ Aggregate v1.4.11
 1. Add a mutex around ODK 1.x form submission creation and updating. This should eliminate submission data corruption.
 2. If only ODK 2.0 privileges are assigned, hide the Submissions tab.
 3. Fix: advisory memcache interaction to delete the entry when rolling back the datastore update.
-4. Add "Tables Super-user" as a configurable user capability. Users with this capability will be able to modify the table-level and row-level privileges on ODK 2.0 rev 206 data tables. These privileges are advisory and do not provide strong access or revision control.
-5. Add "/roles/granted" URL that returns the roles granted to an authenticated username or google account. This will be used by ODK 2.0 rev 206 tools to determine and save the capabilities of the configured user on a device. Those capabilities are then used to apply advisory access controls on the tables and rows.
+4. Add **Tables Super-user** as a configurable user capability. Users with this capability will be able to modify the table-level and row-level privileges on ODK 2.0 rev 206 data tables. These privileges are advisory and do not provide strong access or revision control.
```
Comment (adammichaelwood) [inline aggregate-upgrade.rst]: If this is literal text from a GUI, it should be `:guilabel:`

File: aggregate-upgrade.rst [hunk 4]
```diff
@@ -194,8 +194,8 @@ Aggregate v1.4.11
...
+5. Add **/roles/granted** URL that returns the roles granted to an authenticated username or google account. This will be used by ODK 2.0 rev 206 tools to determine and save the capabilities of the configured user on a device. Those capabilities are then used to apply advisory access controls on the tables and rows.
```
Comment (adammichaelwood) [inline aggregate-upgrade.rst]: If this is a literal URL string, it should be in double backticks.

File: aggregate-use.rst [hunk 5]
```diff
@@ -33,7 +33,7 @@ Under the :guilabel:`Form list` tab, you will see :guilabel:`Add New Form` butto
 .. image:: /img/aggregate-use/add-form.*
    :alt: Image showing add form button.
 
-When you click on it a box will open asking for details of the form. `Form Definition` is required and `Media File(s)` is optional. Choose the .xml file that will be used. You can also choose the appropriate media files for the form.  
+When you click on it a box will open asking for details of the form. *Form Definition* is required and *Media File(s)* is optional. Choose the .xml file that will be used. You can also choose the appropriate media files for the form.
```
Comment (adammichaelwood) [inline aggregate-use.rst]: If this is literal text from a GUI, it should be `:guilabel:`

File: aggregate-use.rst [hunk 6]
```diff
@@ -138,17 +138,17 @@ You can create a single or multiple filters depending on the subset of data you
 
 Various options under this tab can be used as follows:
 
-- Click on :guilabel:`Add Filter` to add filter to the data. In the :guilabel:`Create filter to` dropdown, `Display/Hide` will specify whether you will be selecting data to show or hide and  `Rows/Columns` will specify whether you will be working with the rows or columns of the table. 
+- Click on :guilabel:`Add Filter` to add filter to the data. In the :guilabel:`Create filter to` dropdown, *Display/Hide* will specify whether you will be selecting data to show or hide and  *Rows/Columns* will specify whether you will be working with the rows or columns of the table.
```
Comment (adammichaelwood) [inline aggregate-use.rst]: Menu items in a dropdown are `:menuselection:`

File: aggregate-use.rst [hunk 7]
```diff
@@ -138,17 +138,17 @@ You can create a single or multiple filters depending on the subset of data you
...
 Various options under this tab can be used as follows:
 
-- Click on :guilabel:`Add Filter` to add filter to the data. In the :guilabel:`Create filter to` dropdown, `Display/Hide` will specify whether you will be selecting data to show or hide and  `Rows/Columns` will specify whether you will be working with the rows or columns of the table. 
+- Click on :guilabel:`Add Filter` to add filter to the data. In the :guilabel:`Create filter to` dropdown, *Display/Hide* will specify whether you will be selecting data to show or hide and  *Rows/Columns* will specify whether you will be working with the rows or columns of the table. 
 
 .. image:: /img/aggregate-use/add-filter.*
    :alt: Image showing add filter option.
 
-If you select `Rows` specify a condition you want to apply in the :guilabel:`where` box. 
+If you select *Rows* specify a condition you want to apply in the :guilabel:`where` box.
```
Comment (adammichaelwood) [inline aggregate-use.rst]: Menu items in a dropdown are `:menuselection:`

File: aggregate-use.rst [hunk 8]
```diff
@@ -138,17 +138,17 @@ You can create a single or multiple filters depending on the subset of data you
...
    :alt: Image showing add filter option.
 
-If you select `Rows` specify a condition you want to apply in the :guilabel:`where` box. 
+If you select *Rows* specify a condition you want to apply in the :guilabel:`where` box. 
 
 .. image:: /img/aggregate-use/row-filter.*
    :alt: Image showing row selection.
 
-If you selected `Columns` specify the columns you wish to display or hide in the :guilabel:`titled` box. 
+If you selected *Columns* specify the columns you wish to display or hide in the :guilabel:`titled` box.
```
Comment (adammichaelwood) [inline aggregate-use.rst]: Menu items in a dropdown are `:menuselection:`

File: aggregate-use.rst [hunk 9]
```diff
@@ -510,13 +510,13 @@ Interfaces used during pull actions are:
 
 - OpenRosa Form Discovery API
 - download forms and media files using URLs form discovery
-- `view/submissionList` to obtain a chunk of submission keys
-- `view/downloadSubmission` to download an individual submission
+- *view/submissionList* to obtain a chunk of submission keys
```
Comment (adammichaelwood) [inline aggregate-use.rst]: literal URI strings should use double backticks

File: aggregate-use.rst [hunk 10]
```diff
@@ -510,13 +510,13 @@ Interfaces used during pull actions are:
...
+- *view/downloadSubmission* to download an individual submission
```
Comment (adammichaelwood) [inline aggregate-use.rst]: literal URI strings should use double backticks

File: aggregate-use.rst [hunk 11]
```diff
@@ -510,13 +510,13 @@ Interfaces used during pull actions are:
...
 
 Interfaces used during push actions are:
 
-- `formUpload` to upload a form and its media files to ODK Aggregate
-- `view/submissionList` to obtain a chunk of submission keys   
+- *formUpload* to upload a form and its media files to ODK Aggregate
```
Comment (adammichaelwood) [inline aggregate-use.rst]: literal URI strings should use double backticks

File: aggregate-use.rst [hunk 12]
```diff
@@ -510,13 +510,13 @@ Interfaces used during pull actions are:
...
+- *view/submissionList* to obtain a chunk of submission keys
```
Comment (adammichaelwood) [inline aggregate-use.rst]: literal URI strings should use double backticks

File: briefcase-forms.rst [hunk 13]
```diff
@@ -147,7 +147,7 @@ Export form data
 This command exports form data with id market_prices from Briefcase's storage directory on the :file:`~/Desktop` and write CSV file to :file:`~/Desktop/market_prices.csv`
 
 
-To get help about the command line operation type `java -jar path_to_jar -help`.
+To get help about the command line operation type *java -jar path_to_jar -help*.
```
Comment (adammichaelwood) [inline briefcase-forms.rst]: Inline references to terminal commands use `:command:`
http://docs.opendatakit.org/contributing/#role-command

(Note: The entire command, including options, etc. goes in the `:command:` role...
the `:option:` role is only for talking about options separately.)

File: verify-downloads.rst [hunk 14]
```diff
@@ -13,7 +13,7 @@ Verifying on Linux and Mac
 
 - Open a terminal window.
 - Navigate to the folder containing the file you downloaded. ( cd "path/to/file" )
-- Then type `sha256sum "filename"` on the terminal.
+- Then type **sha256sum "filename"** on the terminal.
```
Comment (adammichaelwood) [inline verify-downloads.rst]: use `:command:`

File: verify-downloads.rst [hunk 15]
```diff
@@ -29,7 +29,7 @@ Verifying on Windows
 
 - Open a cmd window.
 - Navigate to the folder containing the file you downloaded ( cd "path/to/file" ).
-- Then type certUtil -hashfile "filename" SHA256 on your command prompt.
+- Then type **certUtil -hashfile "filename" SHA256** on your command prompt.
```
Comment (adammichaelwood) [inline verify-downloads.rst]: use `:command:`

**Comment by ankita240796** [comment]
@adammichaelwood  Sorry for so many mistakes. I checked this and made the requested changes.

**Pull request merged**